lib/chef/knife/bootstrap_windows_base.rb

```diff
@@ -162,11 +161,6 @@ def self.included(includer)
             :description => "Location of the Chef Client MSI. The default templates will prefer to download from this location. The MSI will be downloaded from chef.io if not provided.",
             :default => ''
 
-          option :install_as_service,
-            :long => "--install-as-service",
-            :description => "Install chef-client as a Windows service",
-            :default => false
```

Comment:
As of Chef 12.18, released 1/23/2017, you can configure the chef-client to run as as scheduled task from the MSI. This is supported in mixlib install (https://github.com/chef/mixlib-install/pull/167) but not yet in knife-windows.

https://blog.chef.io/2017/02/22/running-chef-in-the-background-on-windows/